FindANSAK.cmake

```diff
@@ -105,317 +108,357 @@ For DevStudio Generators, the picture is slightly murkier. See following comment
 
 ############################################################################
 # announce settings if there were any
-if(ANSAK_DIR)
-    message(STATUS "var ANSAK_DIR is defined: ${ANSAK_DIR}")
+if( ANSAK_DIR )
+    message( STATUS "var ANSAK_DIR is defined: ${ANSAK_DIR}" )
```

Comment:
There is no "standard location" on Windows for includes and libraries, so this find_package plug-in will honour an `ANSAK_DIR` value, either from the environment or the cmake command line. In an attempt to make a default which may be growing, this find_package plug-in will honour a `PROGRAM_DATA` value from the same sources, and in the absence, on Windows, of either one, will default to looking in `C:\ProgramData`